build.gradle

```
@@ -276,6 +276,7 @@ project(':cruise-control') {
     implementation "org.apache.zookeeper:zookeeper:${zookeeperVersion}"
     implementation "io.netty:netty-handler:${nettyVersion}"
     implementation "io.netty:netty-transport-native-epoll:${nettyVersion}"
+    implementation "software.amazon.msk:aws-msk-iam-auth:2.0.2"
```

Bump to latest https://github.com/aws/aws-msk-iam-auth/releases

```suggestion
    implementation "software.amazon.msk:aws-msk-iam-auth:2.3.0"
```